wiki/Ranking_Criteria/osu!taiko/fr.md

```diff
@@ -1,43 +1,183 @@
+- ![](/wiki/shared/diff/easy-t.png?20211215) Kantan
+- ![](/wiki/shared/diff/normal-t.png?20211215) Futsuu
+- ![](/wiki/shared/diff/hard-t.png?20211215) Muzukashii
+- ![](/wiki/shared/diff/insane-t.png?20211215) Oni
+- ![](/wiki/shared/diff/expert-t.png?20211215) Inner/Ura Oni
 
-## Conseils
+### Gameplay
 
-Ces informations sont importantes et devraient être suivis par la plupart des mappers. Cependant, ce ne sont pas des règles, ce qui veut dire qu'un mapper peut choisir de les ignorer sous certaines circonstances. Si vous êtes sur le point d'ignorer un conseil, posez vous des questions : "Est-ce que ce que je m'apprête à apporter quelque chose de positif à ma map ?" Soyez certain de ce que vous faites.
+- **Don/Red note :** Un rythme qui peut être frappé avec n'importe quelle touche du Drum Centre (touches par défaut `X`, `C`).
+- **Kat/Blue note :** Un rythme qui peut être frappé avec n'importe quelle touche du Drum Rim (touches par défaut `Z`, `V`).
+- **Big don/Big red note/Finisher note :** Un rythme fort qui peut être frappé avec les deux touches du Drum Centre en même temps pour du score supplémentaire.
+- **Big kat/Big blue note/Finisher note :** Un rythme fort qui peut être frappé avec les deux touches du Drum Rim en même temps pour du score supplémentaire.
+- **BPM :** Acronyme de "battements par minute" utilisé pour déterminer le tempo d'une musique.
+- **Slider/Drumroll :** Une barre jaune avec des ticks à l'intérieur qui sont joués avec n'importe quelle touche assignée. Ces ticks sont généralement réglés sur 1/4 correspondant au BPM de la musique. Si le BPM est inférieur à 125, le snapping passe à 1/8. Si le BPM est supérieur à 250, il passe à 1/2. Si le slider tick rate est réglé sur 3, les sliders ticks seront plutôt réglés sur 1/3.
+- **Spinner :** Un élément rond qui demande au joueur d'alterner entre frapper Don/Kat pour un nombre déterminé de fois, qui est indiqué au centre du spinner. Ce nombre dépend de l'Overall Difficulty de la beatmap ainsi que de la longueur du spinner.
+- **Bar line :** Une ligne qui apparaît sur le playfield et qui détermine le début d'une mesure.
+- **Overlap :** Une note qui recouvre partiellement ou totalement une ou plusieurs autres notes dans le playfield.
+- **Rest moment :** Une période de temps sans notes utilisée spécifiquement pour permettre au joueur de reposer ses mains et de se préparer pour les patterns à venir.
+- **Stream :** Cercles consécutifs regroupés. Habituellement, ils sont frappés à 1/4 de temps.
+- **Snapping :** Tick de la timeline où un objet est placé.
+  - **Variable snapping :** Une combinaison de plusieurs façons différentes de snap les notes dans un court laps de temps en raison de la nature fluctuante de la musique à ce moment-là.
```

Comment:
i don't think i agree with breaking it up for one item, since english has the whole list on the top level

```suggestion
- **Variable snapping :** Une combinaison de plusieurs façons différentes de snap les notes dans un court laps de temps en raison de la nature fluctuante de la musique à ce moment-là.
```